config/routes.rb

```diff
@@ -7,5 +7,5 @@
     resources :discover, only: [:index]
   end
 
-  resources :register, only: [:new, :create], controller: "users"
+  resources :register, only: [:index, :new, :create], controller: "users"
```

Comment:
Why do we need an index for register?

Reply:
oops no, good catch i forgot to take that out